node/peer.js

```diff
@@ -460,7 +409,6 @@ PreferOutgoingHandler.prototype.getQOS = function getQOS() {
     }
 };
 
-// Consulted depending on the peer state
```

Comment:
Hyperbahn is supposed to prefer to speak to services by its outgoing connections, and services are supposed to prefer to speak to Hyperbahn by their incoming connections. This allows Hyperbahn to manage the set of open connections in response to ring membership changes. This feature is far from complete. We still need to implement "drain and close" for wrong-way connections, when those connections are replaced by connections in the preferred direction.